Library/Formula/clang-format.rb

```diff
@@ -0,0 +1,46 @@
+class ClangFormat < Formula
+  homepage "http://clang.llvm.org/docs/ClangFormat.html"
+  version "2014-12-03"
```

Comment:
This is the version that Google uses internally, it's officially called "testing". It's gone through a lot of QA, so it's close in stability to a "stable" version.